### DISCUSSION TITLE
Worked for MSI B365 PRO VDH motherboard

### DISCUSSION THREAD

**Discussion by SantiagoGaonaC** [ixjf/MSIRGB#236] [Q&A]
![image](https://user-images.githubusercontent.com/53282017/130364643-1452f802-e263-434d-a062-d6b9598a18d8.png)

**Comment by ixjf** [comment]
Disable or uninstall all anti-cheats (Riot Vanguard, FACEIT, or others), if you have them.

**Reply by SantiagoGaonaC** [reply]
Yes it works! Thank you very much
